<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>432</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>551</width>
        <height>41</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>47</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Поиск:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineSearch">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>491</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Специальность</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="table"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="buttonAdd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDelCycle">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Удалить цикл</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDisc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>42</height>
           </size>
          </property>
          <property name="text">
           <string>Распред. по
дисциплинам</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>БД</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="action_disconnect"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="action_discipline"/>
    <addaction name="action_book"/>
    <addaction name="action_special"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Документы</string>
    </property>
    <addaction name="action_fond"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Отчет</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Справочники</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Главная таблица</string>
   </property>
  </action>
  <action name="action_fond">
   <property name="text">
    <string>Книжный фонд</string>
   </property>
  </action>
  <action name="action_discipline">
   <property name="text">
    <string>Циклы и дисциплины</string>
   </property>
  </action>
  <action name="action_book">
   <property name="text">
    <string>Книги</string>
   </property>
  </action>
  <action name="action_special">
   <property name="text">
    <string>Специальности</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="text">
    <string>Подключение</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="text">
    <string>Отключение</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
